firefox-ios/Providers/Pocket/PocketProvider.swift

```diff
@@ -9,16 +9,15 @@ import Storage
 protocol PocketStoriesProviding {
     typealias StoryResult = Swift.Result<[PocketFeedStory], Error>
 
-    func fetchStories(items: Int, completion: @escaping (StoryResult) -> Void)
     func fetchStories(items: Int) async throws -> [PocketFeedStory]
 }
 
 extension PocketStoriesProviding {
     func fetchStories(items: Int) async throws -> [PocketFeedStory] {
-        return try await withCheckedThrowingContinuation { continuation in
-            fetchStories(items: items) { result in
-                continuation.resume(with: result)
-            }
+        do {
+            return try await fetchStories(items: items)
+        } catch {
+            throw error
```

Comment:
```suggestion
            return try await fetchStories(items: items)
```
nit: I think I spotted one more! But I'm going to approve anyway. 👍

Reply:
You spoke too soon @ih-codes, I have more changes coming because tests are broken 🤣 (which I had a feeling anyways)

Reply:
Oh no!! 😂  Well you can just re-flag me for review if needed but I'm sure your tests will be great! 👌

Reply:
All done! Its not hard to fix these but had to find time. Please double check in case I missed something in my cleanup 🙈